plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -1509,6 +1509,40 @@ var _ = Describe("validator", func() {
 					Expect(err).To(BeForbiddenError())
 				})
 
+				It("should reject to create a cluster with an expired kubernetes version", func() {
+					deprecatedClassification := core.ClassificationDeprecated
+					expiredKubernetesVersion := "1.18.1"
+					validKubernetesVersion := "1.18.3"
+					shoot.Spec.Kubernetes.Version = expiredKubernetesVersion
+					cloudProfile.Spec.Kubernetes.Versions = append(cloudProfile.Spec.Kubernetes.Versions, core.ExpirableVersion{Version: expiredKubernetesVersion, Classification: &deprecatedClassification, ExpirationDate: &metav1.Time{Time: metav1.Now().Add(time.Second * -1000)}}, core.ExpirableVersion{Version: validKubernetesVersion})
+
+					Expect(coreInformerFactory.Core().InternalVersion().Projects().Informer().GetStore().Add(&project)).To(Succeed())
+					Expect(coreInformerFactory.Core().InternalVersion().CloudProfiles().Informer().GetStore().Add(&cloudProfile)).To(Succeed())
+					Expect(coreInformerFactory.Core().InternalVersion().Seeds().Informer().GetStore().Add(&seed)).To(Succeed())
+					attrs := admission.NewAttributesRecord(&shoot, nil, core.Kind("Shoot").WithVersion("version"), shoot.Namespace, shoot.Name, core.Resource("shoots").WithVersion("version"), "", admission.Create, &metav1.CreateOptions{}, false, nil)
+
+					err := admissionHandler.Admit(context.TODO(), attrs, nil)
+
+					Expect(err).To(BeForbiddenError())
```

Comment:
Nit: You could add `Expect(err.Error()).NotTo(ContainSubstring("< part of error string>"))` for added precision, since most of the error returned in this file are forbidden error.

Reply:
Thanks, done!